docs/source/graph-construction/index.rst

```diff
@@ -2,10 +2,10 @@
 Graph Construction
 ==================
 
-Graphstorm offers various methods to build graphs on both single machines and distributed clusters.
+Graphstorm offers various methods to build graphs on both a single machine and distributed clusters.
 
 .. toctree::
    :maxdepth: 2
    :glob:
 
-   gs-processing/index/gs-processing-index.rst
\ No newline at end of file
+   gs-processing/index.rst
```

Comment:
Do we have a doc for gconstruct? If yes, please also add a ref here.

Reply:
We will have one eventually but including in the following PR. It requires some more doc refactor.

Reply:
OK